src/main/java/org/variantsync/diffdetective/analysis/Analysis.java

```diff
@@ -204,11 +206,16 @@ default void initializeResults(Analysis analysis) {}
         default void beginBatch(Analysis analysis) throws Exception {}
         default boolean beginCommit(Analysis analysis) throws Exception { return true; }
         /**
-         * Signals a parsing failure of some patch in the current commit.
+         * Signals a parsing failure of all patches in the current commit.
          * Called at most once during the commit phase. If this hook is called {@link
          * onParsedCommit} and the following patch phase invocations are skipped.
          */
         default void onFailedCommit(Analysis analysis) throws Exception {}
+        /**
+         * Signals a parsing failure of some patch in the current commit.
+         * Called at most once during the commit phase.
+         */
+        default void onFailedParse(Analysis analysis) throws Exception {}
```

Comment:
I required this Hook in order to capture cases in which only part of a diff could not be parsed. `onFailedCommit` is only called if the entire diff could not be parsed.

Reply:
Looks good to me.